source/includes/steps-general-connection-tab.yaml

```diff
@@ -0,0 +1,66 @@
+---
+title: Click on :guilabel:`Advanced Connection Options`.
+level: 4
+ref: compass-advanced-connection
+content: |
+  .. figure:: /images/compass/advanced-connection-options.png
+     :figwidth: 690px
+     :alt: New Advanced Connection Options
+---
+title: Select :guilabel:`General`.
+level: 4
+ref: select-general-tab
+content: |
+  .. procedure::
+  
+     .. step:: Select a Connection String Scheme.
+
+        You can select from the following two options:
+
+        .. list-table::
+           :header-rows: 1
+           :class: border-table
+
+           * - Connection String Scheme
+             - Definition
+
+           * - mongodb
+             - :manual:`Standard Connection String Format </reference/connection-string/>`. 
+               The standard format of the MongoDB connection URI is used to 
+               connect to a MongoDB deployment: standalone, replica set, or a sharded 
+               cluster. 
+
+           * - mongodb+srv 
+             - DNS Seed List Connection Format. The +srv indicates to the client that 
+               the hostname that follows corresponds to a 
+               :manual:`DNS SRV record </reference/connection-string/>`.
+
+     .. step:: Enter a Hostname.
```

Comment:
There's a little bit of a disconnect because the UI field is `Host`, not `Hostname`. I think the title of this step is fine, but maybe the first sentence of this step should say:

"In the :guilabel:`Host` field, enter the hostname of the machine where the deployment is running."

Reply:
Done.